#### 1 Виконав базові приклади Python у файлі Test.py
#### 2 Результат виконання команди 'python3 .':
    * We are in the __main__
    * 2020-11-08 16:31:53.691655
    * linux
#### 3 Виконуємо 'python3 . -h' і появляється текст з опційними аргументами
        * -h     - допомога
        * -o OPT - ця опція є вибірковою
        * -l     - ввівши цей параметр будуть виведені логи
#### 4 Передавши 'python3 . -o "Цей текст також має вивестись"' наступний параметр, отримуємо додаткове повідомлення від програми:
        * З консолі було передано аргумент
        * ========== >> Цей текст також має вивестись << ==========
#### 5 Якщо ввести python3 . --logs, то будуть виведені усі можливі логи, які прописані всередині програми.
#### 6 Створив  функцію в common.py та викликав її з __main__ з допомогою імпортування модулю.
#### 7  Створив функцію, яка виконується з помилкою та виводить ERROR повідомлення з допомогою бібліотеки logging

# end of Lab2a
